Mean/calMean.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
```

Comment:
Hi, @manit2110 thanks for the PR just a small change, can you remove this line because numpy library is not being used here.